lib/onelogin/ruby-saml/settings.rb

```diff
@@ -71,6 +71,8 @@ def initialize(overrides = {}, keep_security_attributes = false)
       attr_accessor :description
       attr_accessor :privacy_statement_url
       attr_accessor :logo
+      attr_accessor :logo_width
+      attr_accessor :logo_height
```

Comment:
Do we want this change covered in the settings test as well? There's quite a few accessors that aren't covered already.

Reply:
Don't think it is necessary